README.md

```diff
@@ -11,13 +11,16 @@
 ```go
 import "go.linecorp.com/centraldogma"
 
-// create client with OAuth2 token
+// create a client with OAuth2 token
+// See also: https://line.github.io/centraldogma/auth.html#application-token
 centraldogma.NewClientWithToken(baseURL, token, nil)
 ```
 
 ### Customize transport
 
-If transport is `nil` (like above), `http2.Transport` is used by default. You could inject your own transport easily:
+If transport is `nil` (like above), `http2.Transport` is used by default.
+
```

Comment:
nit; could remove this line?

Reply:
I saw in web browser and feel it's more beautiful to add new line :omg: I don't have strong point for this.